### PR TITLE
chore: document commands notation validation

### PR DESCRIPTION
**Pull request by loicbourgois**
Internal ticket: APIGW-1668

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,3 +84,31 @@ Keep in mind only the **pull request title** will be used as commit message as w
 See [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md).
 
 Thank you for reading through all of this, if you have any question feel free to [reach us](README.md#reach-us)!
+
+## Developer documentation
+
+### Commands notation
```
Comment (kindermoumoute) [inline CONTRIBUTING.md]: ==> "When to use dash vs underscore"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -84,3 +84,31 @@ Keep in mind only the **pull request title** will be used as commit message as w
...
+
+In this CLI, we use dash `"-"` for:
```
Comment (kindermoumoute) [inline CONTRIBUTING.md]: nit: remove `In this CLI`

Comment (kindermoumoute) [reply CONTRIBUTING.md]: We SHOULD use dash

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -84,3 +84,31 @@ Keep in mind only the **pull request title** will be used as commit message as w
...
+- command namespaces, resource, verb
+- argument names
+- key names in the human formatted results
+- UUID
+
+In this CLI, we use underscore `"_"` for:
```
Comment (kindermoumoute) [inline CONTRIBUTING.md]: We SHOULD use underscore

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -84,3 +84,31 @@ Keep in mind only the **pull request title** will be used as commit message as w
...
+```bash
+> scw my-namespace my-resource get id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+id               xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+some-key         some_value
+other-key        other_value
+```
+
+### Commands validation
+
+For the namespaces, resources, and verbs, the validation is done in Scaleway's internal `mordor` repository, in `mordor/protoc-gen-mordor/validate.go`.
```
Comment (kindermoumoute) [inline CONTRIBUTING.md]: nit: remote for ==> `For namespaces, resources, ...`

Comment (kindermoumoute) [reply CONTRIBUTING.md]: - Remove `mordor`
-  `the validation is done internally on Scaleway's side`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -84,3 +84,31 @@ Keep in mind only the **pull request title** will be used as commit message as w
...
+
+The rest is manually checked for now as it is too difficult to automate it.
```
Comment (kindermoumoute) [inline CONTRIBUTING.md]: nit: remove

File: CONTRIBUTING.md (hunk 4 above)
Comment (loicbourgois) [reply CONTRIBUTING.md]: add link to sdk-go/docs/continuous-doc-deployment

**Pull request merged**